docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/dockerhub/index.md

```diff
@@ -77,20 +82,22 @@ To configure a Docker Hub repository to send events to the webhook receiver:
 
    1. Provide a name for the webhook.
 
-   1. Set <Hlt>Webhook URL</Hlt> to the value you
-      [retrieved earlier](#retrieving-the-receivers-url).
+   1. Set <Hlt>Webhook URL</Hlt> to the
+      [receiver URL](#retrieving-the-receivers-url).
 
    1. Click <Hlt>+</Hlt> to create webhook.
 
       ![Create Webhook](./img/create-webhook.png "Create Webhook Button")
 
 ## Verifying Webhook Delivery
```

Comment:
Looking more closely at the GitHub example, this whole section seems self-invented and not in line with the format of the other document.